### PR TITLE
Add extra info about windows task-tray program -- 0.16.x

### PR DESCRIPTION
**Pull request by MCGallaspy**
## Summary

Companion to #5007. Rebasing seems to have picked up some extra commits as well.

### CODE REVIEW

File: docs/installguide/release_notes.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ General
 Python 2.6 is no longer supported. It *may* still work, but we are no longer actively supporting it.
 Other known issues:
 
-* The latest OSX version (EL Capitan) is not yet supported.
+* The latest OSX version (EL Capitan) is not yet supported. KA Lite is officially supported on OS X 10.8 - 10.10.
```
Comment (radinamatic) [inline docs/installguide/release_notes.rst]: Isn't El Capitan now supported? 10.11?

Comment (MCGallaspy) [inline docs/installguide/release_notes.rst]: @aronasorman ?

Comment (aronasorman) [inline docs/installguide/release_notes.rst]: Yup. El Capitan is now supported. That's 10.11.

Comment (aronasorman) [inline docs/installguide/release_notes.rst]: But these are 0.15.0 release notes. We should add an "El capitan is now supported" for 0.16.0.

Comment (MCGallaspy) [inline docs/installguide/release_notes.rst]: Okay, I'll amend that here.

Comment (MCGallaspy) [inline docs/installguide/release_notes.rst]: Done.

**Pull request merged**